change/@fluentui-react-charting-d2171258-9397-47d2-b063-9e1b96df4f2d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33525&commitId=f85c863a5db1ff80a1c2be99f9a94c8d1c9471ba&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|5361| Changed|
|react-charting-HeatMapChart.Basic RTL.chromium.png|6720| Changed|
|react-charting-HeatMapChart.Basic.chromium.png|7095| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>